src/locales/en/messages.json

```diff
@@ -1237,6 +1237,18 @@
   "yourVaultIsLockedPinCode": {
     "message": "Your vault is locked. Verify your PIN code to continue."
   },
+  "unlockWithWindowsHello": {
+    "message": "Unlock with Windows Hello"
+  },
+  "windowsHelloConsentMessage": {
+    "message": "Please authenticate yourself to unlock the Bitwarden Vault."
+  },
+  "unlockWithTouchId": {
+    "message": "Unlock with Touch ID"
+  },
+  "touchIdConsentMessage": {
+    "message": "Please authenticate yourself to unlock the Bitwarden Vault."
+  },
```

Comment:
Ony mobile we call this consent message the "direction" and just say "Use your fingerprint to verify." or "Use Face ID to verify.". Should we be consistent here? Something like "Use Windows Hello to verify." and "Use your fingerprint to verify". The consent message may not be displayed int eh context of unlocking the vault.

Reply:
I think we should include Bitwarden in the message since it's just a popup prompt. What about `Verify for Bitwarden.`?

Reply:
That works